package.json

```diff
@@ -12,5 +12,9 @@
   "dependencies": {
     "mustache": "^2.3.0",
     "node-fetch": "^1.7.3"
+  },
+  "devDependencies": {
+    "eslint": "^4.12.1",
+    "eslint-config-prettier": "^2.9.0"
```

Comment:
eslint-config-prettier is for turning off eslint rules that conflict with prettier

https://github.com/prettier/eslint-plugin-prettier should be used instead